custom_mappings.yaml

```diff
@@ -2111,9 +2140,8 @@ entries:
     seasons:
       - season: 1
         anilist-id: 20829
-      - season: 1
+      - season: 2

```

Comment:
This should be kept S1 for both, as only TMDB uses the S1 and S2 naming, see here: https://www.themoviedb.org/tv/62255-seraph-of-the-end/seasons?language=en-US

While TVDB uses single season:
https://thetvdb.com/series/seraph-of-the-end

A solution to your problem would be adding the mapping manually in your custom_mappings.yaml file, since local mappings override the remote ones, that should your best solution.